.github/workflows/test.yml

```diff
@@ -267,15 +269,19 @@ jobs:
 
     - name: Install dependencies (Linux)
       if: ${{ matrix.deps != '' && startsWith(matrix.os, 'ubuntu') }}
-      run: sudo apt update -y && sudo apt install -y ${{ matrix.deps }}
+      run: sudo apt update -y && sudo apt install -y ${{ matrix.deps }} && mkdir ~/.ccache
+
+    - name: Install dependencies (MacOS)
+      if: ${{ matrix.deps != '' && startsWith(matrix.os, 'macos') }}
+      run: brew install ${{ matrix.deps }} && mkdir ~/Library/Caches/ccache
 
     - name: Activate Docker/QEMU
       if: startsWith(matrix.run_under, 'docker')
       run: docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
 
     - name: Set cache key
       if: ${{ matrix.cache }}
-      run: echo "::set-output name=uniq::$(bazel query --output build //external:${{ matrix.repo }} | grep -E 'sha256|commit' | cut -d\" -f2)-$(echo ${{ matrix.flags }} | sha256sum)"
+      run: echo "uniq=$(bazel query --output build //external:${{ matrix.repo }} | grep -E 'sha256|commit' | cut -d\" -f2)-$(echo ${{ matrix.flags }} | sha256sum)" >> $GITHUB_OUTPUT
```

Comment:
Nit: This is unrelated to rest of the changes, could you split it into a separate PR?

Reply:
Could you send a PR with this change to silence the warnings in the CI? Thanks!